demo/vl-form-validation.html

```diff
@@ -130,7 +130,7 @@
                         <p is="vl-form-validation-message" error data-vl-error-id="validation-uuid-error"></p>
                     </div>
                     <div is="vl-form-column" data-vl-size="9" data-vl-push="3">
-                        <button is="vl-button">Formulier valideren</button>
+                        <button id="button-form1" is="vl-button">Formulier valideren</button>
```

Comment:
In andere PRs hebben we ne keer die id's een 'naam' gegeven ipv form-1 en form-2.
Misschien hier ook te bekijken?